src/NeoPixelBrightnessBus.h

```diff
@@ -31,6 +31,37 @@ License along with NeoPixel.  If not, see
 template<typename T_COLOR_FEATURE, typename T_METHOD> class NeoPixelBrightnessBus : 
     public NeoPixelBus<T_COLOR_FEATURE, T_METHOD>
 {
+private:
+    void ConvertColor(typename T_COLOR_FEATURE::ColorObject &color)
```

Comment:
I only allow reference objects that are const to passed in.  If you are defining an in/out argument, it must be passed by address/pointer.

Further, access modifiers like * and & go with the type, not the variable.

`(typename T_COLOR_FEATURE::ColorObject* color)`